eng/slngen.targets

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <SlnGenSolutionFolder Condition="'$(IsGeneratorProject)' == 'true'">gen</SlnGenSolutionFolder>
-    <SlnGenSolutionFolder Condition="'$(IsReferenceAssembly)' == 'true'">ref</SlnGenSolutionFolder>
-    <SlnGenSolutionFolder Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true'">tests</SlnGenSolutionFolder>
+    <SlnGenSolutionFolder Condition="'$(IsReferenceProject)' == 'true'">ref</SlnGenSolutionFolder>
```

Comment:
While there's goodness in a standardized suffix, "IsReferenceProject" feels like it's more about P2P referencing than "a project that produces a reference assembly".  "IsReferenceAssembly" conveyed that better. For a succinct but consistent name "IsReferenceAssemblyProject" seems more appropriate.